tutorials/intro/plot_object_annotations.py

```diff
@@ -1,245 +1,238 @@
+eeglab_raw = mne.io.read_raw_eeglab(eeglab_raw_file)
+print(eeglab_raw.annotations)
 
 ###############################################################################
-# Iterating, indexing and slicing return a copy. This has implications like the
-# fact that changes are not kept.
-
-# this change is not kept
-annot[0]['onset'] = 42
-print(annot[0])
-
-# this change is kept
-annot.onset[0] = 42
-print(annot[0])
-
+# The core data within an :class:`~mne.Annotations` object is accessible
+# through three of its attributes: ``onset``, ``duration``, and
+# ``description``. Here we can see that there were 154 events stored in the
+# EEGLAB file, they all had a duration of zero seconds, there were two
+# different types of events, and the first event occurred about 1 second after
+# the recording began:
+
+print(len(eeglab_raw.annotations))
+print(set(eeglab_raw.annotations.duration))
+print(set(eeglab_raw.annotations.description))
+print(eeglab_raw.annotations.onset[0])
 
 ###############################################################################
-# Save
-# ~~~~
+# More information on working with :class:`~mne.Annotations` objects, including
+# how to add annotations to :class:`~mne.io.Raw` objects interactively, and how
+# to plot, concatenate, load, save, and export :class:`~mne.Annotations`
+# objects can be found in the tutorial :doc:`../raw/plot_annotating_raw`.
 #
-# Note that you can also save annotations to disk in FIF format::
 #
-#     >>> annot.save('my-annot.fif')
+# Converting between Events arrays and Annotations objects
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Or as CSV with onsets in (absolute) ISO timestamps::
+# Once your experimental events are read into MNE-Python (as either an Events
+# array or an :class:`~mne.Annotations` object), you can easily convert between
+# the two formats as needed. You might do this because, e.g., an Events array
+# is needed for epoching continuous data, or because you want to take advantage
+# of the "annotation-aware" capability of some functions, that automatically
+# omit spans of data if they overlap with certain annotations.
 #
-#     >>> annot.save('my-annot.csv')
-#
-# Or in plain text with onsets relative to ``orig_time``::
-#
-#     >>> annot.save('my-annot.txt')
+# To convert an :class:`~mne.Annotations` object to an Events array, use the
+# function :func:`mne.events_from_annotations` on the :class:`~mne.io.Raw` file
+# containing the annotations. This function will assign an integer Event ID to
+# each unique element of ``raw.annotations.description``, and will return the
+# mapping of descriptions to integer Event IDs along with the derived Event
+# array. By default, one event will be created at the onset of each annotation;
+# this can be modified via the ``chunk_duration`` parameter of
+# :func:`~mne.events_from_annotations` to create equally spaced events within
+# each annotation span. See the function documentation for further details.
```

Comment:
```md
See ~the function documentation for further details~ more details in 
:ref:New section in this doc <_bla>`.
```

I would not touch this part, but instead of referring to the API would do a section at the end of this document illustrating `chunk_duration` and `mne.make_fixed_length_events`, and point it from here.

In this new section I would point to the resting state tutos/examples, the functions API and I would let the API point back to this section (ie in the ``chunk_duration``).